package.json

```diff
@@ -15,5 +16,31 @@
   "bugs": {
     "url": "https://github.com/carterwc/Main-Photo-Carousel/issues"
   },
-  "homepage": "https://github.com/carterwc/Main-Photo-Carousel#readme"
+  "homepage": "https://github.com/carterwc/Main-Photo-Carousel#readme",
+  "devDependencies": {
+    "babel-core": "^6.26.3",
+    "babel-loader": "^7.1.5",
+    "babel-preset-env": "^1.7.0",
+    "babel-preset-es2015": "^6.22.0",
+    "babel-preset-react": "^6.23.0",
+    "webpack": "^4.27.1",
+    "webpack-cli": "^3.1.2"
+  },
+  "dependencies": {
+    "axios": "^0.18.0",
+    "body-parser": "^1.18.3",
+    "express": "^4.16.4",
+    "morgan": "^1.9.1",
+    "mysql": "^2.16.0",
+    "nodemon": "^1.18.7",
+    "react": "^16.6.3",
+    "react-dom": "^16.6.3",
+    "request": "^2.88.0",
+    "require": "^2.4.20"
+  },
+  "babel": {
+    "presets": [
+      "es2015"
```

Comment:
use `env`, not `es2015`

Reply:
also you should probably not specify 2 different locations for your babel configs. i would omit this one (in the `package.json`) since you already have babel configs in your `webpack.config.js`